engines/director/detection_tables.h

```diff
@@ -3480,6 +3536,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1_l("sakuratoasobo", "", "Let\'s Play with Sakura", "8d83064402198a909b0abcebc6b46d2b", 118198, Common::JA_JPN, 602),
 	WINGAME1_l("sakuratoasobo", "", "Let\'s Play with Sakura.exe", "45871c12eb944f09f50ee742113a1e2d", 1982850, Common::JA_JPN, 650),
 
+	WINGAME2_l("samson", "", "start.exe", "a593079aecf5bd938ce75264cac24b2d", 1919963,
+								"START.DXR", "ad34b9d4987fc84c238f0e88b174fbc9", 229050, Common::NL_NLD, 600),
```

Comment:
Do you have a better file for the second detection target? E.g. something less generic than "Start.dxr"?